MapboxNavigation/Constants.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+struct MapConstants {
```

Comment:
These should be made public. Although not as pretty, it might be easer to just create variables as seen here https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxCoreNavigation/Constants.swift. I don't think this will bridge to obj c as is.

Reply:
Since these constants are strongly associated with NavigationMapView, you could alternatively make them class properties on NavigationMapView. That’ll give you the namespacing you want in both Swift and Objective-C.